### PR TITLE
open-webui: 0.3.2 -> 0.3.3

### PR DESCRIPTION
**Pull request by drupol**
diff: https://github.com/open-webui/open-webui/compare/v0.3.2..v0.3.3
changelog: https://github.com/open-webui/open-webui/releases/tag/v0.3.3

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by drupol** [comment]
@ofborg build open-webui

**Review by SuperSandro2000 — APPROVED** [review]
Diff lgtm

**Comment by drupol** [comment]
The diff is fine, but the build is not... there's an issue with duplicated `orjson` dependency.

**Comment by drupol** [comment]
@ofborg build open-webui

File: pkgs/development/python-modules/duckduckgo-search/default.nix [hunk 1]
```diff
@@ -26,23 +23,6 @@ let
       };
     }
   );
-
-  orjson_3_10_3 = orjson.overrideAttrs (
```
Comment (natsukium) [inline pkgs/development/python-modules/duckduckgo-search/default.nix:30]: Looks good, but it remains a problem in release-24.05. (https://github.com/NixOS/nixpkgs/pull/310576)
I would also like to remove the `curl-cffi` override and backport them.

Comment (drupol) [reply pkgs/development/python-modules/duckduckgo-search/default.nix:30]: Good idea, job done.

**Comment by natsukium** [comment]
Could you remove `litellm` from dependencies and add `extract_msg` and `pydub` explicitly?
https://github.com/open-webui/open-webui/compare/v0.3.2..v0.3.3#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

Besides this, lgtm.

**Comment by drupol** [comment]
> Could you remove `litellm`

Oh nice catch, thanks! I just fixed it by removing it.

> from dependencies and add `extract_msg` and `pydub` explicitly?

They are already there actually.

**Comment by drupol** [comment]
@ofborg build open-webui

**Comment by drupol** [comment]
@ofborg build open-webui

**Comment by natsukium** [comment]
> They are already there actually.

Yes, it actually works fine for now. I'm not sure which packages propagate them, however, if they stop providing them, `open-webui` will be broken.
I have seen many PRs like "add missing dependencies".

**Comment by drupol** [comment]
> I'm not sure which packages propagate them

I mean, they are in `open-webui` already :)

**Comment by natsukium** [comment]
Oh, sorry, thanks for your clarification.

**Comment by natsukium** [comment]
This PR should targeted to `staging`.🙃 We have to wait for the `ofborg-eval` job at least.
`nixpkgs-review` I started before going to bed still has 3000 steps left.

**Pull request merged**